packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -2053,6 +2053,29 @@ export const ColumnHeaderFocusRingTable = {
   }
 };
 
+export const Blah = {
```

Comment:
? nice name ;)

Reply:
oh whoops lol, test story that I forgot to remove, thanks